src/block-experiments-setup.js

```diff
@@ -1,8 +1,25 @@
+/**
+ * Internal dependencies
+ */
 // This file is to set up the jetpack/layout-grid block that currently lives in block-experiments/blocks/layout-grid
 import { registerBlock } from '../block-experiments/blocks/layout-grid/src';
+import setupLocale from './setup-locale';
+import { getTranslation as getLayoutGridTranslation } from './i18n-translations/layout-grid';
 
-export default function setupBlockExperiments( capabilities ) {
+const LAYOUT_GRID_LOCALE_DOMAIN = 'layout-grid';
```

Comment:
The domain for the Layout Grid block is `layout-grid`, [here](https://github.com/Automattic/block-experiments/blob/00dbb963f1615d9843f0d8a8479b585c7a0108e1/blocks/layout-grid/src/index.js#L45) you can see an example.